src/parser/test/EventRequestParserTest.php

```diff
@@ -1512,6 +1603,104 @@ public function testParseEventRequest()
+            $this->assertTrue($source instanceof UserSource);
+            $this->assertEquals('userid', $source->getUserId());
+            $this->assertInstanceOf(\LINE\Webhook\Model\DeactivatedEvent::class, $event);
+            $this->assertEquals('testwebhookeventid', $event->getWebhookEventId());
+            $this->assertFalse($event->getDeliveryContext()->getIsRedelivery());
+        }
+
+        {
+            // botSuspended
+            $event = $events[41];
+            $this->assertEquals(12345678901234, $event->getTimestamp());
+            $this->assertEquals('active', $event->getMode());
+            $this->assertInstanceOf(\LINE\Webhook\Model\BotSuspendedEvent::class, $event);
+            $this->assertEquals('testwebhookeventid', $event->getWebhookEventId());
+            $this->assertFalse($event->getDeliveryContext()->getIsRedelivery());
+        }
+
+        {
+            // botResumed
+            $event = $events[42];
+            $this->assertEquals(12345678901234, $event->getTimestamp());
+            $this->assertEquals('active', $event->getMode());
+            $this->assertInstanceOf(\LINE\Webhook\Model\BotResumedEvent::class, $event);
+            $this->assertEquals('testwebhookeventid', $event->getWebhookEventId());
+            $this->assertFalse($event->getDeliveryContext()->getIsRedelivery());
+        }
+
+        {
+            // delivery
+            $event = $events[43];
+            $source = $event->getSource();
+            $this->assertEquals(12345678901234, $event->getTimestamp());
+            $this->assertEquals('active', $event->getMode());
+            $this->assertTrue($source instanceof UserSource);
+            $this->assertEquals('userid', $source->getUserId());
+            $this->assertInstanceOf(\LINE\Webhook\Model\PnpDeliveryCompletionEvent::class, $event);
+            $this->assertInstanceOf(\LINE\Webhook\Model\PnpDelivery::class, $event->getDelivery());
+            $this->assertEquals('deliverydata', $event->getDelivery()->getData());
+            $this->assertEquals('testwebhookeventid', $event->getWebhookEventId());
+            $this->assertFalse($event->getDeliveryContext()->getIsRedelivery());
+        }
+
+        {
+            // module (attached)
+            $event = $events[44];
+            $this->assertEquals(12345678901234, $event->getTimestamp());
+            $this->assertEquals('active', $event->getMode());
+            $this->assertInstanceOf(\LINE\Webhook\Model\ModuleEvent::class, $event);
+            /** @var \LINE\Webhook\Model\AttachedModuleContent $module */
+            $module = $event->getModule();
+
+            $actualClassName = get_class($module);
+
+            $this->assertInstanceOf(\LINE\Webhook\Model\AttachedModuleContent::class, $module);
+            $this->assertInstanceOf(\LINE\Webhook\Model\AttachedModuleContent::class, $module, "Expected 'LINE\Webhook\Model\AttachedModuleContent', got '$actualClassName'");
```

Comment:
This fails even though debug(error) log in `parseModuleContent` showed its class is `AttachedModuleContent`. How should I do ...?

Reply:
It's resolved by https://github.com/line/line-bot-sdk-php/pull/556/commits/228f0442f7d1cc5fcb82b9db0cd33de41e2c4054. Thank you @moririnson san!!!!